modules/schema/pages/procedure-lot-part-information.adoc

```diff
@@ -1883,13 +1882,42 @@ within that Award Criteria no Award Criterion with Award Criterion Number Weight
 </cac:AwardingCriterion>
 ----
 
-When Award Criteria calculation is too complex to be described with the proposed structure, 
+When Award Criteria calculation is too complex to be described with the 
+proposed structure including Award Criterion Number (BT-541), 
 the complicated expression, or a reference to the procurement document section where this 
 information may be found, should be specified in Award Criteria Complicated (BT-543). 
 Information is encoded as text with the element "_cbc:CalculationExpression_". For a given 
 Award Criteria (i.e. "_cac:AwardingCriterion_" element), the children "_cbc:Description_" 
 (Award Criteria Order Justification, BT-733) and "_cbc:CalculationExpression_" (Award 
-Criteria Complicated, BT-543) shall not coexist.
+Criteria Complicated, BT-543) shall not coexist. Similarly, for a given Award 
+Criteria (i.e. "_cac:AwardingCriterion_" element), the child element 
+"_cbc:CalculationExpression_" (Award Criteria Complicated, BT-543) and the 
+descendant element "_efbc:ParameterNumeric_" (Award Criterion Number, BT-541) 
+shall not coexist.
+
+
+[source,xml]
+----
+<cac:AwardingCriterion>
+    <cbc:CalculationExpression languageID="ENG">Price-quality score calculation is based on ...</cbc:CalculationExpression>
+    <cbc:CalculationExpression languageID="FRA">Le calcul du score prix-qualité est basée sur ...</cbc:CalculationExpression>
+    <cac:SubordinateAwardingCriterion>
+        <cbc:AwardingCriterionTypeCode listName="award-criterion-type">price</cbc:AwardingCriterionTypeCode>
+        <cbc:Name languageID="ENG">Price</cbc:Name>
+        <cbc:Name languageID="FRA">Prix</cbc:Name>
+        <cbc:Description languageID="ENG">The price score is calculated by ...</cbc:Description>
+        <cbc:Description languageID="FRA">Le score de prix est calculé par ...</cbc:Description>
+    </cac:SubordinateAwardingCriterion>
+    <cac:SubordinateAwardingCriterion>
+        <cbc:AwardingCriterionTypeCode listName="award-criterion-type">quality</cbc:AwardingCriterionTypeCode>
+        <cbc:Name languageID="ENG">Quality</cbc:Name>
+        <cbc:Name languageID="FRA">Qualité</cbc:Name>
+        <cbc:Description languageID="ENG">The quality score is calculated by ...</cbc:Description>
+        <cbc:Description languageID="FRA">Le score de qualité est calculé par ...</cbc:Description>
+    </cac:SubordinateAwardingCriterion>
+</cac:AwardingCriterion>
+----
```

Comment:
I would also have kept the other example which is a bit different as it may be difficult to catch how all this works

Reply:
? I didn't remove any examples. I only removed cbc:CalculationExpression from the first example as it had the Numbers.

Reply:
Thx